### PR TITLE
Added the monthly community meeting, meeting notes, and code of conduct

### PR DESCRIPTION
**Pull request by mtwo**

### CODE REVIEW

File: content/community/_index.md [hunk 1]
```diff
@@ -15,3 +15,9 @@ Any questions or suggestions? You can reach us via email: [census-developers@goo
 
 #### Gitter Channel
 Join other developers and users on the OpenCensus [Gitter channel](https://gitter.im/census-instrumentation/Lobby).
+
+#### Community Meetings
+We hold community meetings once per month, and they're open to all contributors. Meeting invites can be accessed from our [public Google Calendar](https://calendar.google.com/calendar?cid=Z29vZ2xlLmNvbV9iNzllM2U5MGo3YmJzYTJuMnA1YW41bGY2MEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t), and we keep public notes in the agenda document available [here](https://docs.google.com/document/d/1sG3MAwHzYTlv5PrIBBpjZBTiShOpmzmPlLRe8NXZKnM/edit).
+
+#### Code of Conduct
```
Comment (odeke-em) [inline content/community/_index.md]: @mtwo I already made https://github.com/census-instrumentation/opencensus-website/pull/454 and I made it a top-level page not hidden under community. I'll get that merged then you can rebase.

File: content/community/_index.md [hunk 2]
```diff
@@ -15,3 +15,9 @@ Any questions or suggestions? You can reach us via email: [census-developers@goo
 
 #### Gitter Channel
 Join other developers and users on the OpenCensus [Gitter channel](https://gitter.im/census-instrumentation/Lobby).
+
+#### Community Meetings
+We hold community meetings once per month, and they're open to all contributors. Meeting invites can be accessed from our [public Google Calendar](https://calendar.google.com/calendar?cid=Z29vZ2xlLmNvbV9iNzllM2U5MGo3YmJzYTJuMnA1YW41bGY2MEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t), and we keep public notes in the agenda document available [here](https://docs.google.com/document/d/1sG3MAwHzYTlv5PrIBBpjZBTiShOpmzmPlLRe8NXZKnM/edit).
```
Comment (odeke-em) [inline content/community/_index.md]: Nice, this is very important, thanks @mtwo!

File: content/conduct/_index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Code of Conduct"
+draft: false
+weight: 110
+---
+
```
Comment (odeke-em) [inline content/conduct/_index.md]: Right here let's add this
```

- [Our pledge](#our-pledge)
- [Our Standards and Values](#our-standards-and-values)
- [Our Responsibilities](#our-responsibilities)
- [Scope](#scope)
- [Enforcement](#enforcement)
- [Attribution](#attribution)
```
and a new line as a clickable table of content

Comment (mtwo) [reply content/conduct/_index.md]: On it

Comment (mtwo) [reply content/conduct/_index.md]: Added, though I'm guessing there's a better way. Can you take a look?

Comment (odeke-em) [reply content/conduct/_index.md]: LGTM!

**Pull request merged**